pyshelf/configure.py

```diff
@@ -9,26 +9,32 @@ def app(app, config_path):
         content = f.read()
         config = yaml.load(content)
 
-    if not config.get("buckets"):
-        raise ValueError("config.yaml requires buckets heading for list of buckets with associated keys.")
+    _validate_key("buckets", config)
```

Comment:
Ah, this is getting out of hand.  I'm going to write something for schema validation